### PR TITLE
Check in ci-scheduling-webhook configs

### PR DESCRIPTION
**Pull request by bear-redhat**
Work done by @jupierce

https://github.com/openshift/ci-tools/pull/3807

### CODE REVIEW

**Review by hongkailiu — COMMENTED** [review]
Are the files in `clusters/ci-scheduling-webhook` going to be applied to some clusters?

**Comment by bear-redhat** [comment]
> Are the files in clusters/ci-scheduling-webhook going to be applied to some clusters?
I symlinked.

File: clusters/ci-scheduling-webhook/dns.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# https://github.com/openshift/enhancements/blob/master/enhancements/dns/configurable-dns-pod-placement.md#proposal
+# dns-default-* pods do not tolerate all taints. This means our infra nodes and nodes tainted to support
+# pod classes are not getting these pods scheduled onto them.
+# This seemed to lead to https://bugzilla.redhat.com/show_bug.cgi?id=2086887 .
+apiVersion: operator.openshift.io/v1
```
Comment (aleskandro) [inline clusters/ci-scheduling-webhook/dns.yaml:5]: Hi, this is breaking CI for BM QE when resolving names as conflicting with https://github.com/openshift/release/blob/master/clusters/build-clusters/build05/assets/dns.yaml

Can we merge them for build05?

**Pull request merged**